tmail-backend/rate-limiter/rate-limiter-postgres/src/main/scala/com/linagora/tmail/rate/limiter/api/postgres/repository/PostgresRateLimitingPlanUserRepository.scala

```diff
@@ -0,0 +1,47 @@
+package com.linagora.tmail.rate.limiter.api.postgres.repository
+
+import com.google.common.base.Preconditions
+import com.linagora.tmail.rate.limiter.api.postgres.dao.PostgresRateLimitPlanUserDAO
+import com.linagora.tmail.rate.limiter.api.{RateLimitingPlanId, RateLimitingPlanNotFoundException, RateLimitingPlanUserRepository}
+import javax.inject.Inject
+import org.apache.james.backends.postgres.utils.PostgresExecutor
+import org.apache.james.core.Username
+import org.reactivestreams.Publisher
+import reactor.core.publisher.Mono
+import reactor.core.scala.publisher.SMono
+
+class PostgresRateLimitingPlanUserRepository @Inject()(executorFactory: PostgresExecutor.Factory) extends RateLimitingPlanUserRepository {
+  private val daoWithoutRLS: PostgresRateLimitPlanUserDAO =
+    PostgresRateLimitPlanUserDAO(executorFactory.create())
```

Comment:
use non-rls connection

Reply:
It is needed. Please read the code more to know why.

Reply:
Very good catch indeed. Thanks @hungphan227